python/example01.py

```diff
@@ -0,0 +1,149 @@
+import pycurl
+import os
+import sys
+import json
+import io
+
+AK_SIZE = 20
+SK_SIZE = 40
+
+class Tag:
+    def __init__(self, resources_id, key, value, resources_type):
+        self.resources_id = resources_id
+        self.key = key
+        self.value = value
+        self.resources_type = resources_type
+
+def setup_curl(url, json_data, ak_sk):
```

Comment:
why not, but this function doesn't only setup curl, but setup_curl and send the request. 
I would prefer to have to function: 1 to setup curl, called one in the main, and one to send the request, call every times.